src/WebJobs.Script.WebHost/Management/InstanceManager.cs

```diff
@@ -75,12 +98,13 @@ public bool StartAssignment(HostAssignmentContext context)
 
                 // all assignment settings/files have been applied so we can flip
                 // the switch now on specialization
+                _logger.LogInformation("Triggering specialization");
```

Comment:
This relies on SpecializationTimer tick which runs every second to actually switch modes right? Wonder if we should more logs to see how long cold start is delayed by the timer frequency.